ores/scoring_context.py

```diff
@@ -34,18 +34,13 @@ def __init__(self, name, model_map, extractor):
         self.update(model_map)
         self.extractor = extractor
 
-    def format_model_info(self, model_names, fields=None):
-        fields = fields or ['version']
+    def format_model_info(self, model_name, fields=None):
         model_info = {}
-        for model_name in model_names:
-            formatted_info = self._get_model_info_for(model_name)
-            filtered_info = {field: value
-                             for field, value in formatted_info.items()
-                             if fields == "all" or field in fields}
-
-            model_info[model_name] = filtered_info
-
-        return model_info
+        formatted_info = self._get_model_info_for(model_name)
+        filtered_info = {field: value
+                         for field, value in formatted_info.items()
+                         if fields == [""] or field in fields}
```

Comment:
I'd like to see the `fields == [""]` test extracted into a is_all_fields predicate...  It looks arch-magical in there.